components/forms/select/single/component.tsx

```diff
@@ -218,7 +217,7 @@ export const SingleSelect: FC<SelectProps> = ({
               {getOptions.map((option, index) => (
                 <li
                   className={cx({
-                    'px-4 py-1 mt-0.5 cursor-pointer': true,
+                    'px-2 py-1 mt-0.5 cursor-pointer text-tiny font-bolder': true,
```

Comment:
This should be all uppercase, also the selected one

Reply:
Also increase the y padding a bit

Reply:
padding y is 4px which is also same in design.